diff --git a/debian/changelog b/debian/changelog
index 4d06618..8e31481 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,3 +1,9 @@
+lightfield (1.0.1) stable; urgency=medium
+
+  * First update.
+
+ -- LightField packager <lightfield-packager@volumetricbio.com>  Mon, 17 Jun 2019 14:12:21 -0700
+
 lightfield (1.0.0) stable; urgency=medium
 
   * Initial Release.
diff --git a/make-deb-package.sh b/make-deb-package.sh
index dfea097..cdc3aac 100755
--- a/make-deb-package.sh
+++ b/make-deb-package.sh
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-VERSION=1.0.0
+VERSION=1.0.1
 PACKAGE_BUILD_ROOT=/home/lumen/Volumetric/LightField/packaging
 
 #########################################################
diff --git a/make-upgrade-kit.sh b/make-upgrade-kit.sh
index f4e2c59..1017197 100755
--- a/make-upgrade-kit.sh
+++ b/make-upgrade-kit.sh
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-VERSION=1.0.0
+VERSION=1.0.1
 PACKAGE_BUILD_ROOT=/home/lumen/Volumetric/LightField/packaging
 
 #########################################################
diff --git a/src/version.cpp b/src/version.cpp
index cd5b791..abfa4ef 100644
--- a/src/version.cpp
+++ b/src/version.cpp
@@ -2,10 +2,10 @@
 
 #include "version.h"
 
-char      const* LIGHTFIELD_VERSION_STRING     = "1.0.0";
+char      const* LIGHTFIELD_VERSION_STRING     = "1.0.1";
 unsigned  const  LIGHTFIELD_VERSION_MAJOR      = 1;
 unsigned  const  LIGHTFIELD_VERSION_MINOR      = 0;
-unsigned  const  LIGHTFIELD_VERSION_TEENY      = 0;
+unsigned  const  LIGHTFIELD_VERSION_TEENY      = 1;
 unsigned  const  LIGHTFIELD_VERSION_CODE       = MakeVersionCode( LIGHTFIELD_VERSION_MAJOR, LIGHTFIELD_VERSION_MINOR, LIGHTFIELD_VERSION_TEENY );
 
 #if defined _DEBUG
diff --git a/unpack-kit-manually.sh b/unpack-kit-manually.sh
index 89d2310..b66a627 100755
--- a/unpack-kit-manually.sh
+++ b/unpack-kit-manually.sh
@@ -11,7 +11,7 @@ UPDATES_DIR=/var/lib/lightfield/software-updates
 rm    -v -dr "${UPDATES_DIR}"
 mkdir -v -p  "${UPDATES_DIR}"
 
-for ver in 1.0.0
+for ver in 1.0.1
 do
 	dir="lightfield-debug_${ver}_amd64"
 	updatedir="${UPDATES_DIR}/${dir}"
@@ -22,7 +22,7 @@ done
 
 chown -c -R  lumen:lumen "${UPDATES_DIR}"
 
-echo "deb file:/var/lib/lightfield/software-updates/lightfield-debug_1.0.0_amd64 ./" > /etc/apt/sources.list.d/volumetric-lightfield.list
+echo "deb file:/var/lib/lightfield/software-updates/lightfield-debug_1.0.1_amd64 ./" > /etc/apt/sources.list.d/volumetric-lightfield.list
 chown -c lumen:lumen /etc/apt/sources.list.d/volumetric-lightfield.list
 
 cp    -v ~/volumetric-keyring.gpg /etc/apt/trusted.gpg.d/
